sql/parse/parse.go

```diff
@@ -109,15 +111,39 @@ func convertSet(ctx *sql.Context, n *sqlparser.Set) (sql.Node, error) {
 			return nil, err
 		}
 
-		fmt.Println(strings.TrimSpace(strings.ToLower(e.Name.Qualifier.Name.String())))
-		switch strings.TrimSpace(strings.ToLower(e.Name.Qualifier.Name.String())) {
-		case "@@session", "": // do nothing
-		default:
-			return nil, ErrUnsupportedFeature.New("qualifiers in set variable names other than @@session")
+		name := strings.TrimSpace(e.Name.Lowered())
+		if strings.Contains(name, "autocommit") {
```

Comment:
Is ON/OFF only for autocommit on MySQL?

Reply:
Maybe there are different cases (I didn't investigate any other cases, just focused on this particular case), but the main issue here can be transformation. For some expressions `Type()` panics.
I'm totally open to have the most generic solution transformation here.